File: problems/n/nastyhacks/nastyhacks.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())

    for _ in range(n):
        r, e, c = [int(val) for val in input().split()]
        
        if r == (e - c):
            print('does not matter')
        else:
            if r > (e - c):
                print('do not advertise')
            else:
                print('advertise')

if __name__ == '__main__':
    main()

File: problems/n/nastyhacks/nastyhacks.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int n, r, e, c;
    if (scanf("%d", &n) != 1) {
        printf("Failed to read n!");
    }

    for (int i=0; i<n; i++) {
        if (scanf("%d%d%d", &r, &e, &c) != 3) {
            printf("Failed to read r, e and c!");
        }
        
        if (r == (e - c)) {
            printf("does not matter\n");
        }
        else if (r > (e - c)) {
            printf("do not advertise\n");
        }
        else {
            printf("advertise\n");
        }

    }

    return 0;
}